elements/pfe-card/src/pfe-card.scss

```diff
@@ -193,11 +221,12 @@ $variables: (
 
     @each $tag in (h1, h2, h3, h4, h5, h6) {
       ::slotted(#{$tag}) {
-        margin-bottom: 0;
+        margin-bottom: 0 !important; // Helps overcome standard browser, bootstrap, or normalize styles
```

Comment:
I just put an issue in the issue queue for this on pfe-tabs, should this go somewhere global like pfe-sass?

https://github.com/patternfly/patternfly-elements/issues/745

Reply:
like if pfe-sass had this:
```
@mixin heading-clear {
    @each $tag in (h1, h2, h3, h4, h5, h6) {
      ::slotted(#{$tag}) {
        margin-bottom: 0 !important; // Helps overcome standard browser, bootstrap, or normalize styles
      }
   }
}
```

Reply:
I love it! Maybe the default list is headings but you could pass anything into it? `margin-clear` would more open ended for that purpose.

Reply:
I added a clear-spacing mixin in the last commit: https://github.com/patternfly/patternfly-elements/pull/592/commits/b6f196cfe99343e35f8855dfb88c898f29a2d221